src/Mod/TechDraw/App/DrawViewBalloon.h

```diff
@@ -52,8 +52,8 @@ class TechDrawExport DrawViewBalloon : public TechDraw::DrawView
     App::PropertyLink        SourceView;
     App::PropertyString      Text;
     App::PropertyEnumeration EndType;
-    App::PropertyEnumeration Symbol;
-    App::PropertyFloatConstraint SymbolScale;
+    App::PropertyEnumeration Shape;

```

Comment:
If you change the property name, doesn't this break older files that were saved with the previous name?

Reply:
You are right, stupid me.

Reply:
This is now fixed: af5007c